packages/gmime.rb

```diff
@@ -1,6 +1,6 @@
-require 'package'
+require 'buildsystems/autotools'
 
-class Gmime < Package
+class Gmime < Autotools
   description 'GMime is a powerful MIME (Multipurpose Internet Mail Extension) utility library. It is meant for creating, editing, and parsing MIME messages and structures.'
   homepage 'https://developer.gnome.org/gmime/'
   version '3.2.3'
```

Comment:
Update to [3.2.14](https://github.com/jstedfast/gmime/releases/tag/3.2.14)?

Reply:
Does not build.